wrappers/services_gen_test.go

```diff
@@ -262,6 +267,7 @@ apps:
         stop-timeout: 10s
         bus-name: foo.bar.baz
         daemon: dbus
+        activates-on: [dbus-slot]
```

Comment:
should we keep a test where only bus-name is specified?

Reply:
Good idea.  I've added an extra test case.